msteams-platform/resources/schema/manifest-localization.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Localization manifest schema reference
+description: Describes the localization schema supported by the localization manifest for Microsoft Teams
+keywords: teams manifest schema localization
+ms.date: 05/20/2019
+---
+
+# Reference: Localization manifest schema for Microsoft Teams
+
+The Microsoft Teams localization manifest describes language translations that will be served based on the client language settings. Your file must conform to the schema hosted at [`https://developer.microsoft.com/en-us/json-schemas/teams/v1.5/MicrosoftTeams.Localization.schema.json`]( https://developer.microsoft.com/en-us/json-schemas/teams/v1.5/MicrosoftTeams.Localization.schema.json). For additional information see [app localization](~/publishing/apps-localization.md).
```

Comment:
I'd rather not call this a 'manifest' can change wording: Microsoft Teams localization manifest -> Microsoft Teams localization strings file?

Reply:
@akinolugbade Thoughts on this one? I'm not really fond of using "manifest" either. Do you have a preference?